hdk-rust/src/api.rs

```diff
@@ -340,8 +341,35 @@ pub fn remove_entry<S: Into<String>>(_entry: HashString, _message: S) -> ZomeApi
 }
 
 /// Not Yet Available
-pub fn get_links<S: Into<String>>(_base: HashString, _tag: S) -> ZomeApiResult<Vec<HashString>> {
-    Err(ZomeApiError::FunctionNotImplemented)
+pub fn get_links<S: Into<String>>(base: &HashString, tag: S) -> ZomeApiResult<Vec<HashString>> {
+    let mut mem_stack = unsafe { G_MEM_STACK.unwrap() };
+
+    // Put args in struct and serialize into memory
+    let input = GetLinksArgs {
+        entry_address: base.clone(),
+        tag: tag.into(),
+    };
+
+    let allocation_of_input = store_as_json(&mut mem_stack, input)
+        .map_err(|err_code| ZomeApiError::Internal(err_code.to_string()))?;
+
+    let encoded_allocation_of_result: u32 =
+        unsafe { hc_get_links(allocation_of_input.encode() as u32) };
+
+    // Deserialize complex result stored in memory and check for ERROR in encoding
+    let result: GetLinksResult = load_json(encoded_allocation_of_result as u32)
+        .map_err(|err_str| ZomeApiError::Internal(err_str))?;
+
+    // Free result & input allocations and all allocations made inside commit()
+    mem_stack
+        .deallocate(allocation_of_input)
+        .expect("deallocate failed");
+
+    if result.ok {
+        Ok(result.links)
```

Comment:
I think this should be just like get_entry() in that it should return the result (i.e. the LinkEntriesResult object) not the links out of the result because the result will end up having the meta data (crud status,sources) and so-on, so we should have that be the the same starting from now.  It's just a tiny change.

Reply:
Ok, will change this now.